src/hdmf/term_set.py

```diff
@@ -0,0 +1,91 @@
+from collections import namedtuple
```

Comment:
I'm wondering whether we should move `ExternalResources`, `ExternalResourceMangager`, `TermSet` etc. codes into a separate module (folder) for external_resources. I just think it would be easiest to keep those things in one place. However, that would break with the logic of having things defined in the common schema in the `hdmf.common` folder. Maybe have a module (folder) in `hdmf.common`, e.g,. `hdmf.common.external_resources`. @rly what do you think?